### PR TITLE
Changelog for 0.26.0

### PR DESCRIPTION
**Pull request by vrubezhny**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
 # Change Log
 
+## [0.26.0](https://github.com/eclipse/lemminx/milestone/42?closed=1) (June 15, 2023)
+
+### Enhancements
+
+ * TextDocument to expose an API for getting a document line number by offset. See [#1519](https://github.com/eclipse/lemminx/pull/1519).
```
Comment (rgrunber) [inline CHANGELOG.md:7]: I would reword this as : Expose API in `TextDocument` for getting a document line number from its offset.

For things coming from the code base I occasionally will use backticks (eg. ` ) to surround them.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
...
+ * Make ICodeActionParticipant members default. See [#1518](https://github.com/eclipse/lemminx/issues/1518).
```
Comment (rgrunber) [inline CHANGELOG.md:8]: I would reword this as : Add a `default` implementation for `ICodeActionParticipant` members.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+ * Allow code actions to be created if no Diagnostics is provided. See [#1516](https://github.com/eclipse/lemminx/issues/1516).
```
Comment (rgrunber) [inline CHANGELOG.md:9]: I would reword this as : Add support for code actions that are not associated with any diagnostics (quick assist).

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
+ * IRenameParticipant.doRename() is not conform with LSP-defined rename result code action. See [#1521](https://github.com/eclipse/lemminx/issues/1521).
```
Comment (rgrunber) [inline CHANGELOG.md:10]: I would reword this as : Make the rename participants conform to the LSP, permitting to rename across different documents.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,24 @@
...
+
+### Bug Fixes
+
+ * xml.foldings.includeClosingTagInFold has no effect in binary mode. See [#1523](https://github.com/eclipse/lemminx/issues/1523).
```
Comment (rgrunber) [inline CHANGELOG.md:14]: I would just add back-ticks to `xml.foldings.includeClosingTagInFold`

**Review by rgrunber — CHANGES_REQUESTED** [review]
Overall looks fine. Just a few small changes. Do you want to release today or is that just a placeholder ?

@datho7561 @angelozerr . I would probably prepare the vscode-xml release next week at best.

File: CHANGELOG.md (hunk 1 above)
Comment (vrubezhny) [reply CHANGELOG.md:7]: Ack

File: CHANGELOG.md (hunk 2 above)
Comment (vrubezhny) [reply CHANGELOG.md:8]: Ack

File: CHANGELOG.md (hunk 3 above)
Comment (vrubezhny) [reply CHANGELOG.md:9]: Ack

File: CHANGELOG.md (hunk 4 above)
Comment (vrubezhny) [reply CHANGELOG.md:10]: Ack

File: CHANGELOG.md (hunk 5 above)
Comment (vrubezhny) [reply CHANGELOG.md:14]: Ack

**Comment by vrubezhny** [comment]
PR is updated according to @rgrunber 's notes

**Comment by vrubezhny** [comment]
> Overall looks fine. Just a few small changes. Do you want to release today or is that just a placeholder ?

@rgrunber If possible and no other changes are planned,  releasing today would be a good thing.  Otherwise, please feel free to change the PR as needed.  or just ping me when ready to start releasing and I'll update the release date.

**Review by datho7561 — APPROVED** [review]

**Comment by vrubezhny** [comment]
@rgrunber @datho7561 Thanks for the review!
